### ISSUE TITLE
Names that are too long generate invalid multi-line strings

### ISSUE THREAD

**Issue by danielxvu** [arttor/helmify#12]
The following snippet of YAML results in an invalid credentials.yaml file:

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: spellbook-credentials
type: Opaque
data:
  MQTT_HOST: "Zm9vYmFy"
  MQTT_PORT: "Zm9vYmFy"
  ELASTIC_HOST: "Zm9vYmFy"
  ELASTIC_PORT: "Zm9vYmFy"
  ELASTIC_PROTOCOL: "Zm9vYmFy"
  ELASTIC_PASSWORD: "Zm9vYmFy"
  ELASTIC_FOOBAR_HUNTER123_MEOWTOWN_VERIFY: "Zm9vYmFy"
```
The resulting output:

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: {{ include "spellbook.fullname" . }}-credentials
  labels:
  {{- include "spellbook.labels" . | nindent 4 }}
data:
  ELASTIC_FOOBAR_HUNTER123_MEOWTOWN_VERIFY: {{ required "credentials.elasticFoobarHunter123MeowtownVerify
    is required" .Values.credentials.elasticFoobarHunter123MeowtownVerify | b64enc
    | quote }}
  ELASTIC_HOST: {{ required "credentials.elasticHost is required" .Values.credentials.elasticHost
    | b64enc | quote }}
  ELASTIC_PASSWORD: {{ required "credentials.elasticPassword is required" .Values.credentials.elasticPassword
    | b64enc | quote }}
  ELASTIC_PORT: {{ required "credentials.elasticPort is required" .Values.credentials.elasticPort
    | b64enc | quote }}
  ELASTIC_PROTOCOL: {{ required "credentials.elasticProtocol is required" .Values.credentials.elasticProtocol
    | b64enc | quote }}
  MQTT_HOST: {{ required "credentials.mqttHost is required" .Values.credentials.mqttHost
    | b64enc | quote }}
  MQTT_PORT: {{ required "credentials.mqttPort is required" .Values.credentials.mqttPort
    | b64enc | quote }}
```

Running `helm install` will error out because of the invalid YAML generated for key names past a certain length:

```console
$ helm install -f ./contrib/k8s/examples/spellbook/values.yaml webrtc-dev-telemetry ./contrib/k8s/charts/spellbook
Error: INSTALLATION FAILED: parse error at (spellbook/templates/credentials.yaml:8): unterminated quoted string
```

Thanks for your work on this, by the way.

**Comment by yxd-ym** [comment]
I wrote a tool to sanitize templates generated by helmify.

https://github.com/yxd-ym/go-template-sanitizer

Just
``` bash
cat your-template.yaml | go-template-sanitizer
```

**Comment by vicentefb** [comment]
@danielxvu is your required error `{{ required "credentials.elasticHost is required" .Values.credentials.elasticHost
    | b64enc | quote }}` due to the same error? Or does this error has a different source issue?

**Comment by danielxvu** [comment]
@vicentefb No. This issue is caused by an overly long key like ELASTIC_FOOBAR_HUNTER123_MEOWTOWN_VERIFY.

**Comment by gprossliner** [comment]
I'm having the same issue on a secret object. @arttor are you still working on this?

**Comment by arttor** [comment]
Fixed in [0.3.22 release](https://github.com/arttor/helmify/releases/tag/v0.3.22).

**Comment by gprossliner** [comment]
Thank you!
